keepassxc-browser/background/client.js

```diff
@@ -176,6 +185,11 @@ keepassClient.getNonce = function() {
     return nacl.util.encodeBase64(nacl.randomBytes(keepassClient.keySize));
 };
 
+// Creates a short random data for Request ID
+keepassClient.getRequestId = function() {
+    return nacl.util.encodeBase64(nacl.randomBytes(keepassClient.requestIdSize));
+};
```

Comment:
Can we just generate a random number instead of random data? Between 0 and INT_MAX, saves the base64 encoding. Alternative would be a UUID.

Reply:
Sure.